docs/linux_kernel_reporting_bugs.md

```diff
@@ -3,16 +3,62 @@
-Please also add `syzkaller@googlegroups.com` to the CC list.
+To find out the list of maintainers responsible for a particular kernel subsystem, use the [get_maintainer.pl](https://github.com/torvalds/linux/blob/master/scripts/get_maintainer.pl) script: `./scripts/get_maintainer.pl -f guilty_file.c`. Please add `syzkaller@googlegroups.com` to the CC list.
+Make sure to mention the exact kernel branch and revision where the bug occured.
+Many kernel mailing lists reject HTML formatted messages, so use the plain text mode when sending the report.
 
 Bugs without reproducers are way less likely to be triaged and fixed.
 If the bug is reproducible, include the reproducer (C source if possible, otherwise a syzkaller program) and the `.config` you used for your kernel.
 If the reprocucer is available only in the form of a syzkaller program, please link [the instructions on how to execute them](executing_syzkaller_programs.md) in your report.
 Check that the reproducer works if you run it manually.
+Syzkaller tries to simplify the reproducer, but the result might not be ideal.
+You can try to simplify or annotate the reproducer manually, that greatly helps kernel developers to figure out why the bug occurs.
 
-Make sure to also mention the exact kernel branch and revision.
+If you want to get extra credit, you can try to undestand the bug and develop a fix yourself.
+If you can't figure out the right fix, but have some understanding of the bug, please add your thoughts and conclusions to the report, that will save some time for kernel developers.
 
-Many kernel mailing lists reject HTML formatted messages, so use the plain text mode when sending the report.
+## Reporting security bugs
+
+If you believe that a found bug poses potential security threat, consider following the instructions below.
+Note, that these instructions are a work-in-progress and based on my current undestanding of the disclosure proccess.
+
+The three main mailing lists for reporting and disclosing Linux kernel security issues are `security@kernel.org`, `linux-distros@vs.openwall.org` and `oss-security@lists.openwall.com`.
+The guidelines for these lists can be found here:
+
+1. `security@kernel.org` - https://www.kernel.org/doc/html/latest/admin-guide/security-bugs.html
+2. `linux-distros@vs.openwall.org` - http://oss-security.openwall.org/wiki/mailing-lists/distros
+3. `oss-security@lists.openwall.com` - http://oss-security.openwall.org/wiki/mailing-lists/oss-security
+
+### Reporting minor security bugs
+
+To report minor security bugs (such as local DOS or local info leak):
+
+1. Report the bug publicly to kernel developers as described above and wait until a fix is committed. Alternatively, you can develop and send a fix yourself.
+2. Request a CVE from MITRE through [the web form](https://cveform.mitre.org/). Describe the bug details and add a link the fix in the request.
```

Comment:
s/link the fix/link to fix/

link to the commit?

how does one give a link to a kernel commit?